lib/sequelize.js

```diff
@@ -94,6 +95,8 @@ class Sequelize {
    * @param {Array}    [options.retry.match] Only retry a query if the error matches one of these strings.
    * @param {Integer}  [options.retry.max] How many times a failing query is automatically retried.  Set to 0 to disable retrying on SQL_BUSY error.
    * @param {Boolean}  [options.typeValidation=false] Run built in type validators on insert and update, e.g. validate that arguments passed to integer fields are integer-like.
+   * @param {Object}   [options.operators] String aliases for operators if not set all aliases are enabled by default.
```

Comment:
`* @param {Object|Boolean}   [options.operators=true] String based operator alias, default value is true which will enable all operators alias. Pass object to limit set of aliased operators or false to disable completely.`

@yonjah Now I think about it, may be `operatorAlias` is better name after all :)

Reply:
@sushantdhiman singular `operatorAlias` or `operatorsAliases` ?

Reply:
@yonjah  `operatorsAliases`